setup.py

```diff
@@ -17,6 +17,21 @@
 
 import setuptools
 
+# Disable version normalization performed by setuptools.setup()
+# Including this workaround for Python2.7 support
+try:
+    # Try the approach of using sic(), added in setuptools 46.1.0
+    from setuptools import sic
+except ImportError:
+    # Try the approach of replacing packaging.version.Version
+    sic = lambda v: v
+    try:
+        # setuptools >=39.0.0 uses packaging from setuptools.extern
+        from setuptools.extern import packaging
+    except ImportError:
+        # setuptools <39.0.0 uses packaging from pkg_resources.extern
+        from pkg_resources.extern import packaging
+    packaging.version.Version = packaging.version.LegacyVersion
```

Comment:
Remove work around and pin to setuptools version here:
https://github.com/googleapis/python-api-core/blob/master/setup.py#L36

Reply:
My mistake. This is needed for python 2 support.